rl_algorithms/common/networks/brain.py

```diff
@@ -54,3 +55,148 @@ def calculate_fc_input_size(self, state_dim: tuple):
         x = torch.zeros(state_dim).unsqueeze(0)
         output = self.backbone(x).detach().view(-1)
         return output.shape[0]
+
+
+class GRUBrain(Brain):
+    """Class for holding backbone, GRU, and head networks."""
+
+    def __init__(
+        self, backbone_cfg: ConfigDict, head_cfg: ConfigDict,
+    ):
+        self.action_size = head_cfg.configs.output_size
+        """Initialize. Generate different structure whether it has CNN module or not."""
+        super(GRUBrain, self).__init__(backbone_cfg, head_cfg)
```

Comment:
Change to `Brain.__init__(...)` for consistency.

Reply:
Resolved in commit 5ac6d88.